apps/studio/components/interfaces/Auth/BasicAuthSettingsForm/BasicAuthSettingsForm.tsx

```diff
@@ -89,9 +89,12 @@ const BasicAuthSettingsForm = () => {
   const canUpdateConfig = useCheckPermissions(PermissionAction.UPDATE, 'custom_config_gotrue')
 
   const organization = useSelectedOrganization()
-  const { data: subscription, isSuccess: isSuccessSubscription } = useOrgSubscriptionQuery({
-    orgSlug: organization!.slug,
-  })
+  const { data: subscription, isSuccess: isSuccessSubscription } = useOrgSubscriptionQuery(
+    {
+      orgSlug: organization?.slug,
+    },
+    { enabled: !!organization }
```

Comment:
What about `enabled: IS_PLATFORM`?

Underlying RQ won't/shouldn't fire when org slug is undefined either way

Reply:
We can actually move the IS_PLATFORM check to the RQ itself, but would suggest doing that in a separate PR (can look into it eventually)

Reply:
Good idea 👍🏼